tests/query/parser/test_resolver_visitor.py

```diff
@@ -0,0 +1,154 @@
+from typing import Mapping
+
+import pytest
+
+from snuba.query.expressions import (
+    Column,
+    CurriedFunctionCall,
+    Expression,
+    FunctionCall,
+    Lambda,
+    Literal,
+)
+from snuba.query.parser import AliasExpanderVisitor
+
+TEST_CASES = [
+    pytest.param(
+        Column("a", None, "a"),
+        {"b": FunctionCall("b", "f", tuple())},
+        False,
+        Column("a", None, "a"),
+        id="Simple Column - do nothing",
+    ),
+    pytest.param(
+        Literal(None, "a"),
+        {},
+        False,
+        Literal(None, "a"),
+        id="Simple Literal - do nothing",
+    ),
+    pytest.param(
+        Column(None, None, "ref"),
```

Comment:
I don't have a very strong opinion here, and this is ultimately a stylistic call so I'll leave it up to your judgement.

I think that my specific issues are mostly around that I still haven't internalized that the alias is read first so I find myself having to read these multiple times to interpret the structure correctly and in those cases the explicit `None` make it seems to imply the value has more importance than it does in practice.

Reply:
I see what you mean now (as of 63f7c9f) and I think you were right. 😬

Reply:
I see the point of the aliases being mostly None (which is kind of the goal here since I am testing alias references). I turned it into key value arguments. Hopefully we will be able to make it less verbose after getting rid of aliases in the expressions entirely.